### PR TITLE
lightning: add clean up metadata schema

### PR DESCRIPTION
**Pull request by glorv**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-lightning/tidb-lightning-faq.md [hunk 1]
```diff
@@ -191,6 +191,19 @@ upload-speed-limit = "100MB"
 
 3. 如果需要的话，删除 TiDB 集群上创建的所有表和库。
 
+4. 清理残留的元信息。对于 v5.1.x 和 v5.2.x 版本的 tidb-lightning, tidb-lightning-ctl 命令没有同时清理存储在目标集群的 metadata 库，需要手动清理。如果是使用这些版本的 tidb-lightning 或者手动删除断点文件，则需要手动清理下游的元信息库：
```
Comment (sunzhaoyang) [inline tidb-lightning/tidb-lightning-faq.md:194]: 这个描述希望再清晰些？
1. “没有同时清理存储在目标集群的 metadata 库”。这个“同时”是指任务退出时 or 其他的情况？
2. 什么情况下会手动删除断点文件？

Comment (glorv) [reply tidb-lightning/tidb-lightning-faq.md:194]: 这个 faq 描述的是 **手动** 清理残留的 checkpoint, 那显然是任务异常失败的情况。

Comment (hfxsd) [inline tidb-lightning/tidb-lightning-faq.md:194]:
```suggestion
4. 清理残留的元信息。如果存在以下任意一种情况，需要手动清理元信息库：
    
    - 对于 v5.1.x 和 v5.2.x 版本的 TiDB Lightning, tidb-lightning-ctl 命令没有同时清理存储在目标集群的 metadata 库，需要手动清理。
    - 如果手动删除过断点文件，则需要手动清理下游的元信息库，否则可能影响后续导入的正确性。

使用下面命令清理元信息：
```

File: tidb-lightning/tidb-lightning-faq.md [hunk 2]
```diff
@@ -191,6 +191,19 @@ upload-speed-limit = "100MB"
...
+{{< copyable "sql" >}}
+
+```sql
+DROP DATABASE IF EXISTS `lightning_metadata`;
+```
+
+> **注意：**
+> 
+> 在清理断点文件时，请务必清理残留的元信息库，否则有可能影响后续导入的正确性。
+>
```
Comment (hfxsd) [inline tidb-lightning/tidb-lightning-faq.md:205]:
```suggestion
```

File: tidb-lightning/tidb-lightning-faq.md [hunk 3]
```diff
@@ -191,6 +191,20 @@ upload-speed-limit = "100MB"
 
 3. 如果需要的话，删除 TiDB 集群上创建的所有表和库。
 
+4. 清理残留的元信息。如果存在以下任意一种情况，需要手动清理元信息库：
+    
+    - 对于 v5.1.x 和 v5.2.x 版本的 TiDB Lightning, tidb-lightning-ctl 命令没有同时清理存储在目标集群的 metadata 库，需要手动清理。
+    - 如果手动删除过断点文件，则需要手动清理下游的元信息库，否则可能影响后续导入的正确性。
+
+使用下面命令清理元信息：
```
Comment (hfxsd) [inline tidb-lightning/tidb-lightning-faq.md:199]:
```suggestion
    使用下面命令清理元信息：
```

File: tidb-lightning/tidb-lightning-faq.md [hunk 4]
```diff
@@ -191,6 +191,20 @@ upload-speed-limit = "100MB"
...
+    - 如果手动删除过断点文件，则需要手动清理下游的元信息库，否则可能影响后续导入的正确性。
+
+    使用下面命令清理元信息：
+
+{{< copyable "sql" >}}
+
+```sql
+DROP DATABASE IF EXISTS `lightning_metadata`;
+```
+
```
Comment (hfxsd) [inline tidb-lightning/tidb-lightning-faq.md:206]:
```suggestion
    使用下面命令清理元信息：

    {{< copyable "sql" >}}

    ```sql
    DROP DATABASE IF EXISTS `lightning_metadata`;
    ```

```

**Review by hfxsd — APPROVED** [review]
LGTM

**Review by sunzhaoyang — APPROVED** [review]

File: tidb-lightning/tidb-lightning-faq.md [hunk 5]
```diff
@@ -191,6 +191,20 @@ upload-speed-limit = "100MB"
...
+    - 如果手动删除过断点文件，则需要手动清理下游的元信息库，否则可能影响后续导入的正确性。
+
+    使用下面命令清理元信息：
+
+    {{< copyable "sql" >}}
+
+    ```sql
+    DROP DATABASE IF EXISTS `lightning_metadata`;
+    ```
+
```
Comment (hfxsd) [inline tidb-lightning/tidb-lightning-faq.md:206]:
```suggestion
```

**Pull request merged**